test/TestConstants.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace WebJobs.Extensions.DurableTask.Tests
```

Comment:
All source files need to include the copyright header.

Reply:
I will add it shortly.  Thank you!